packages/vm/src/imports.rs

```diff
@@ -35,6 +37,8 @@ const MAX_LENGTH_CANONICAL_ADDRESS: usize = 32;
 /// The maximum allowed size for bech32 (https://github.com/bitcoin/bips/blob/master/bip-0173.mediawiki#bech32)
 const MAX_LENGTH_HUMAN_ADDRESS: usize = 90;
 const MAX_LENGTH_QUERY_CHAIN_REQUEST: usize = 64 * KI;
+/// Max length for a debug message
+const MAX_LENGTH_DEBUG: usize = 2 * MI;
```

Comment:
That's some serious DEBUG output. But sure... stack traces and full storage dumps if we need it.
This is not enabled in the blockchain and not persisted to disk, so I see no trouble having it huge.

Reply:
Yeah I think it does not harm and avoid unnecessary interruptions of a development workflow